src/lib/compress-fiddle.ts

```diff
@@ -31,7 +31,7 @@ export const defaultFiddle: Fiddle = {
 
 class Main {
 	public static void main(String[] args) {
-		System.out.println("Hello, World!"); 
+		System.out.println("Hello, World!");
```

Comment:
I'd prefer if this change wasn't part of the functional removing iframes commit, as it might generate difficulty reading back history.